test/test_dists.jl

```diff
@@ -101,28 +182,24 @@ w = rand(size(a))
 
 # Test KL, Renyi and JS divergences
 p = r = rand(12)
-p[p .< 0.3] = 0.0
+p[p .< median(p)] = 0.0
 scale = sum(p) / sum(r)
 r /= sum(r)
 p /= sum(p)
 q = rand(12)
 q /= sum(q)
 
-klv = 0.
+klv = 0.0
 for i = 1 : length(p)
     if p[i] > 0
         klv += p[i] * log(p[i] / q[i])
     end
 end
 @test kl_divergence(p, q) ≈ klv
 
-@test renyi_divergence(p, p, 0) ≈ 0
```

Comment:
Because they are all tested in the Premetric test - Premetrics are non-negative and zero when measuring a distance from something to itself.